_topic_maps/_topic_map.yml

```
@@ -441,6 +441,8 @@ Topics:
     File: installing-vsphere-installer-provisioned-customizations
   - Name: Installing a cluster on vSphere with network customizations
     File: installing-vsphere-installer-provisioned-network-customizations
+  - Name: Installing a cluster on vSphere with Mutliple Failure Domains
```

```suggestion
  - Name: Installing a cluster on vSphere with multiple failure domains
```
typo and capitalization